app/controllers/application_controller.rb

```diff
@@ -31,6 +32,21 @@ def authorize(record, query = nil, user = nil)
     end
   end
 
+  def check_disabled_client
+    if client_disabled?
+      ex = NotAuthorizedError.new("Client is disabled")
+      fail ex
+    end
+  end
+
+  def render_disabled_client_message(message)
+    begin
+      render "#{current_user.client_slug}/_disabled", status: 403
+    rescue
```

Comment:
would it be possible to rescue a specific type of error? http://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby